chapter_3/CMakeLists.txt

```diff
@@ -11,7 +11,9 @@ include_directories("${PROJECT_SOURCE_DIR}/libflandmark")
 FILE(COPY "haarcascade_frontalface_alt.xml" DESTINATION ${CMAKE_BINARY_DIR})
 FILE(COPY "flandmark_model.dat" DESTINATION ${CMAKE_BINARY_DIR})
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -stdlib=libc++")
+set(CMAKE_C_COMPILER "/usr/bin/clang")
+set(CMAKE_CXX_COMPILER "/usr/bin/clang++")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -stdlib=libc++ -stdlib=libstdc++")
```

Comment:
Just tried with the following flags and with deleted CMAKE_*_COMPILER directives:

`set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")`

and it works for me.

According to the docs, it's necessary to set using C++11 explicitly as there are some directives in the code that require it (e.g.: `to_string(img_id)`)